index.js

```diff
@@ -20,17 +21,54 @@ const releaseOldLockedPortsIntervalMs = 1000 * 15;
 // Lazily create interval on first use
 let interval;
 
-const getAvailablePort = options => new Promise((resolve, reject) => {
-	const server = net.createServer();
-	server.unref();
-	server.on('error', reject);
-	server.listen(options, () => {
-		const {port} = server.address();
-		server.close(() => {
-			resolve(port);
+const getLocalHosts = () => {
+	const interfaces = os.networkInterfaces();
+	const results = new Set();
+
+	for (const _interface of Object.values(interfaces)) {
+		for (const config of _interface) {
+			results.add(config.address);
+		}
+	}
+
+	// Add undefined value for createServer function to use default host,
+	// and default IPv4 host in case createServer defaults to IPv6.
+	results.add(undefined);
```

Comment:
Shouldn't it try the default host first? Meaning, this should be in the `new Set` contractor.

Reply:
ack, makes sense.